ocs_ci/ocs/cluster.py

```diff
@@ -903,18 +903,20 @@ def delete_blockpool(self, pool_name):
         if pool_name == "ocs-storagecluster-cephblockpool":
             return
 
+        # To make the deletion time faster, delete the created pool brutally
+        patch = (
+            f"cephblockpool {pool_name} --type=merge -p "
+            '\'{"metadata":{"finalizers":null}}\''
+        )
```

Comment:
are we deleting this in ceph backend? If so do take care of left over PVCs and PVs.
If it's deleted in order then won't be a problem.

Reply:
there will be no leftover, all PVC / PV are deleted before the pool is deleted.